tools/api-builder/angular.io-package/templates/api-list-audit.template.html

```diff
@@ -0,0 +1,15 @@
+[{% for module, items in doc.data %}
+  {% for item in items %}
+    {
+      "title": "{$ item.title $}",
+      "path": "{$ item.exportDoc.path $}",
+      "docType": "{$ item.docType $}",
+      "stability": "{$ item.stability $}",
+      "secure": "{$ item.security $}",
+      "howToUse": "{$ item.howToUse $}",
```

Comment:
@ericjim @naomiblack : `howToUse` is prose, right? Do you actually want to dump it all here or just have an "Exists" / "Not Done" status like for `whatItDoes`?